web/src/engine/websites/DayComics.ts

```diff
@@ -38,34 +35,60 @@ const matureCookieScript = `
     });
 `;
 
+const libraryScript = `
+    [...document.querySelectorAll('section#libraryPage a[href*="/content/"')].map(anchor => {
+      const link = anchor.pathname;
```

Comment:
no need for intermediate variable if they are only used once => assign the values directly in the return object